### DISCUSSION TITLE
How to make a release to a gitlab private repository ?

### DISCUSSION THREAD

**Discussion by sgaunet** [goreleaser/goreleaser#2072] [Q&A] [answered]
Hello, 

I got a private repo on gitlab with sub folders, the url is like : https://gitlab.com/subrepo1/subrepo2/subrepo3/MyRepo
I'm trying to use goreleaser and I've got this error : 

`release failed after 2.06s error=github/gitlab/gitea releases: failed to publish artifacts: GET https://gitlab.com/api/v4/projects/subrepo3/MyRepo/releases/0.3: 404 {message: 404 Project Not Found}
`
You see, there is not all the sup folders, don't know if it's a bug or a misconfiguration. 
Do you have any idea ?

Thanks a lot

**Comment by caarlos0** [comment]
Check https://github.com/goreleaser/goreleaser/issues/1799

**Reply by mavogel** [reply]
Hi, which gitlab version are you using? 
I will most probably finish https://github.com/goreleaser/acceptance-tests/pull/1 this weekend and can probably give you a `config.yaml` you can reuse. Feel free to take [here](https://gitlab.com/mavogel/release-testing/-/blob/master/goreleaser-wo-docker-local.yml#L5) as well. It was the base repo before.

**Comment by sgaunet** [comment]
I'm late...

```
goreleaser version 0.143.0
commit: a92774f602331c53cd6b2c3c3ec8ec9058a26a7f
built at: 2020-09-06T14:53:58Z
built by: goreleaser
```

I will update to the latest, check again.

**Comment by sgaunet** [comment, marked as answer]
Thank you for your help, very appreciate.

It's ok with the last release 0.157 with the configuration below :

```
release:
  gitlab:
    owner: subrepo1/subrepo2/subrepo3
    name: MyRepo
```

If contributors see this, thank you so much for this tool.

**Comment by sgaunet** [comment]
Unfortunetly, the workaround is not working anymore. Maybe gitlab update its API :'(
Tried the last release of goreleaser without success, any idea ?

**Reply by mavogel** [reply]
hi @sgaunet can you make an issue/new discussion and please provider the versions of 
- goreleaser
- gitlab
This will help us reproduce the issue. TY
